src/ua/zhdanova/hw16/Person.java

```diff
@@ -0,0 +1,51 @@
+package ua.zhdanova.hw16;
+
+
+import java.lang.reflect.Field;
+class Person {
+    private String name;
+    private int age;
+    private String address;
+    public Person(String name, int age, String address) {
+        this.name = name;
+        this.age = age;
+        this.address = address;
+    }
+    public String getName() {
+        return name;
+    }
+    public void setName(String name) {
+        this.name = name;
+    }
+    public int getAge() {
+        return age;
+    }
+    public void setAge(int age) {
+        this.age = age;
+    }
+    public String getAddress() {
+        return address;
+    }
+    public void setAddress(String address) {
+        this.address = address;
+    }
+}
+class PersonFactory {
+    public Person createPerson(String name, int age, String address) {
+        return new Person(name, age, address);
+    }
```

Comment:
Here, we should use reflection api to create an object.
For this, you should create parametrized contructor in Person class and HERE using reflection, get this contructor and create an object.

https://jenkov.com/tutorials/java-reflection/constructors.html

<img width="986" alt="image" src="https://github.com/MariaSukhetska/ALevelHomework/assets/72043323/e1f90dbc-5401-4e1f-bc71-722a1d6ef176">